cmake/developer_package/plugins/plugins.hpp.in

```diff
@@ -21,3 +26,16 @@ inline const std::map<Key, Value> getStaticPluginsRegistry() {
 @IE_PLUGINS_MAP_DEFINITION@
     return plugins_hpp;
 }
+
+#else
+
+using Key = std::string;
+using Value = std::string;
+using PluginsCompiledRegistry = std::map<Key, Value>;
+
+inline const std::map<Key, Value> getCompiledPluginsRegistry() {
+@IE_PLUGINS_COMPILED@
```

Comment:
why do we still need to have a conditional compilation? 
what if I used `ENABLE_PLUGINS_XML` here?